packages/connect/src/connect-s3.ts

```diff
@@ -41,8 +48,7 @@ export class ConnectS3 extends Connection {
       body: JSON.stringify(crdtEntry),
     })
     const result = await done.json()
-    if (result.status != 201)
-    {
+    if (result.status != 201) {
```

Comment:
good to remove reformat-only changes for something like this

Reply:
For every file I always do a format document and then save it. However these stuff keeps happening